templates/portworx/1/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+portworx:
+  labels:
+    io.rancher.container.dns: 'true'
+    io.rancher.container.create_agent: 'true'
+    io.rancher.scheduler.global: 'true'
+    io.rancher.container.pull_image: 'always'
+  image: portworx/px-dev:edge
```

Comment:
Please, use a concrete docker version. Generic labels are not reproducible.
Pull image always is not needed.
```
-    io.rancher.container.pull_image: 'always'
-  image: portworx/px-dev:edge
+  image: portworx/px-dev:1.2.11.1
```
Include docker version in catalog version.